src/Utils/Hooks/useImagePerformanceObserver.ts

```diff
@@ -28,8 +34,8 @@ export const useImagePerformanceObserver = () => {
             entry.initiatorType !== "img" ||
             // Ensure they are uncached
             entry.transferSize === 0 ||
-            // Ensure they are Gemini images
-            !entry.name.includes(GEMINI_CLOUDFRONT_URL) ||
+            // Ensure they come from image domains we want to track
+            !TRACKING_URL_ALLOWLIST.some(url => entry.name.includes(url)) ||
```

Comment:
`includes` is likely a bit permissive over `startsWith` but /shrug

Reply:
OHHH